src/spikeinterface/core/numpyextractors.py

```diff
@@ -517,7 +522,8 @@ def __init__(self, snippets_list, spikesframes_list, sampling_frequency, nbefore
         )
 
         self._is_dumpable = False
```

Comment:
I thought that "dumpable" was mean to be pickable? Can you remind me of how they are different?

Reply:
yes or no.
dumpable is pickle for multiprocessing but absolutly want to avoid to dump silently NumpyRecording when extract waveforms.
`check_serializablility('pickle)` is clearly for the file version.

NumpyRecording is really the corner case we need to have in mind.